qkeras/quantizers.py

```diff
@@ -1742,11 +1728,14 @@ def __call__(self, x):
       p = K.sigmoid(x) * m
```

Comment:
To mirror the quantized_tanh, I suggest setting the `p` variable in the same way you set the `p` variable in `quantized_tanh`:
```
p = K.sigmoid(x) if self.use_real_sigmoid else _sigmoid(x)
```

This may require some other small adjustments in this call function.